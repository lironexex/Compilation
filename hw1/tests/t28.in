// comments 8 - nested comments 1 (note that technically the 'nested' comments in the following cases are just considered plain text)

// this is legal: // note that we are still inside the original comment
// */ this is also legal */
/* // this is legal too */
/* // and this
// and this 
*/

// /* note that this is just considered plain text!
*/
