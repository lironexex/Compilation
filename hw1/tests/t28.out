1 COMMENT 1
3 COMMENT 1
4 COMMENT 1
5 COMMENT 1
8 COMMENT 3
10 COMMENT 1
11 BINOP *
11 BINOP /
